chai/chai.d.ts

```diff
@@ -25,6 +28,28 @@ declare module Chai {
         (target: any, message?: string): Assertion;
     }
 
+    interface ShouldAssertion {
+        equal(val1: any, val2: any, msg?: string): void;
```

Comment:
I can change that (I was just going straight from the [source code](https://github.com/chaijs/chai/blob/2.3.0/lib/chai/interface/should.js#L63), didn't stop to think if it was well worded)